src/components/TodoContainer/TodoContainer.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import TodoList from '../TodoList/TodoList';
+import styles from './TodoContainer.module.css';
+import PropTypes from 'prop-types';
+import AddTodoForm from '../AddTodoForm/AddTodoForm';
+import { getAllTodoItems, deleteTodoItem } from '../../utils/fetchUtil';
+import { sortByTitle, sortByIsChecked } from '../../utils/sortUtil';
+
+function TodoContainer({ tableName }) {
+  const [todoList, setTodoList] = React.useState([]);
+  const [isLoading, setIsLoading] = React.useState(true);
+  const [sort, setSort] = React.useState(false);
+
+  async function loadTodoList() {
+    setIsLoading(true);
+    const todos = await getAllTodoItems();
+    console.log('todo list response:', todos);
+    let sortedList = sortByIsChecked(todos);
+    setTodoList(sortedList);
+    setIsLoading(false);
+  }
+
+  React.useEffect(() => {
+    if (!isLoading) {
+      localStorage.setItem('savedTodoList', JSON.stringify(todoList));
+    }
+  }, [todoList, isLoading]);
+
+  React.useEffect(() => {
+    loadTodoList();
+  }, []);
```

Comment:
This a little detail I noticed in the rubric. 
```suggestion
  React.useEffect(() => {
    loadTodoList();
  }, [tableName]);
```